src/get-started/quickstart.md

```diff
@@ -53,6 +53,20 @@ MinimalWordCount.java	WordCount.java
 
 For a detailed introduction to the Beam concepts used in these examples, see the [WordCount Example Walkthrough]({{ site.baseurl }}/get-started/wordcount-example). Here, we'll just focus on executing `WordCount.java`.
 
+## Runner-Specific Dependencies
```

Comment:
As of now, if you follow the Quickstart steps you will end up with a pom that only has the direct runner and dataflow runner as dependencies. (This is tracked in https://issues.apache.org/jira/browse/BEAM-905) I think we either have to make BEAM-905 a blocker for this or add in specific instructions.